lib/SetAPI/generic/SetInterfaceV1.py

```diff
@@ -26,56 +60,64 @@ def __init__(self: "SetInterfaceV1", workspace_client: Workspace) -> None:
         :param workspace_client: workspace client
         :type workspace_client: Workspace
         """
-        self.ws = workspace_client
+        self.ws_client = workspace_client
 
     def save_set(
         self: "SetInterfaceV1",
-        set_type: str,
+        set_item_name: str,
         provenance: dict[str, Any],
         params: dict[str, Any],
     ) -> dict[str, Any]:
         """Save a set to the workspace as the set type provided.
 
         :param self: this class
         :type self: SetInterfaceV1
-        :param set_type: set type, e.g. KBaseSets.AssemblySet
-        :type set_type: str
+        :param set_item_name: name of the objects in the set, e.g. "genome"
+        :type set_item_name: str
         :param provenance: provenance
         :type provenance: dict[str, Any]
         :param params: parameters for the request
         :type params: dict[str, Any]
         :return: newly-created set item
         :rtype: dict[str, Any]
         """
-        self._check_save_set_params(params)
+        set_item_name = self._check_save_set_params(set_item_name, params)
```

Comment:
slight change here - this method now returns the set_item_name (which the next line converts into the KBase set type) after doing the various validation checks. This is to ensure that the KBaseSearch.GenomeSets get saved as the correct type.